docs-ref-conceptual/install-azure-cli-windows.md

```diff
@@ -35,110 +30,37 @@ The MSI and ZIP distributable are used for installing or updating the Azure CLI
 > [!IMPORTANT]
 > After the installation is complete, you will need to **close and reopen any active terminal window to use the Azure CLI**.
 
-# [Microsoft Installer (MSI)](#tab/azure-cli)
-
-### Latest version
-
-Download and install the latest release of the Azure CLI. When the installer asks if it can make changes to your computer, select the "Yes" box.
-
-> [!div class="nextstepaction"]
-> [Latest MSI of the Azure CLI (32-bit)](https://aka.ms/installazurecliwindows)
-
-> [!div class="nextstepaction"]
-> [Latest MSI of the Azure CLI (64-bit)](https://aka.ms/installazurecliwindowsx64)
-
-If you have previously installed the Azure CLI, running either the 32-bit or 64-bit MSI will overwrite an existing installation.
-
-### Specific version
-
-If you prefer, you can download a specific version of the Azure CLI by using a URL.
-
-[!INCLUDE [specific version](includes/specific-version.md)]
-
-# [Microsoft Installer (MSI) with PowerShell](#tab/powershell)
-
-### PowerShell
-
-To install the Azure CLI using PowerShell, start PowerShell **as administrator** and run the following command:
-
-   ```PowerShell
-   $ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest -Uri https://aka.ms/installazurecliwindows -OutFile .\AzureCLI.msi; Start-Process msiexec.exe -Wait -ArgumentList '/I AzureCLI.msi /quiet'; Remove-Item .\AzureCLI.msi
-   ```
-
-This will download and install the latest 32-bit installer of the Azure CLI for Windows. If you prefer a 64-bit install, change URL to `https://aka.ms/installazurecliwindowsx64`. If the Azure CLI is already installed, the installer will overwrite the existing version.
-
-To install a specific version, replace the `-Uri` argument with the URL described in [Specific version](#specific-version-1). Here is an example of using the 32-bit installer of the Azure CLI version [2.51.0](/cli/azure/release-notes-azure-cli#august-01-2023) in PowerShell:
+::: zone pivot="winget"
```

Comment:
I don't think we should make WinGet the first choice as WinGet's Azure CLI is not maintained by us.